package.json

```diff
@@ -11,15 +11,15 @@
     "watch-poll": "npm run watch -- --watch-poll",
     "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
     "prod": "npm run production",
-    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
+    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
   },
   "devDependencies": {
-    "axios": "^0.16.2",
-    "buefy": "^0.5.4",
-    "bulma": "^0.5.3",
-    "cross-env": "^5.0.1",
+    "axios": "^0.17",
+    "bulma": "^0.5.4",
+    "cross-env": "^5.1",
+    "buefy": "^0.5.4",        
```

Comment:
I've watched all youtube playlist and play around with the code. It have a small change from bulma 0.5.4 to 0.6.1. I would recommend you change to the 0.6.1. Please Check and tell me if i'm wrong @carusogabriel